apps/web/src/components/Shared/Username.tsx

```diff
@@ -0,0 +1,21 @@
+import formatHandle from '@lib/formatHandle';
+import type { Profile } from 'lens';
+import Link from 'next/link';
+import type { FC } from 'react';
+
+import Slug from './Slug';
+
+interface Props {
+  profile: Profile;
+  className?: string;
+}
+
+const UserName: FC<Props> = ({ profile, className }) => {
```

Comment:
can we rename `UserName` to `Username` 🙇🏼

Reply:
done!